reflectable/lib/reflectable.dart

```diff
@@ -39,10 +37,10 @@ abstract class ReflectableInterface {
   /// it can be obtained from `originalDeclaration`. That mirror will
   /// have no actual type arguments, but it will have declared type
   /// variables. Other types than classes are not (yet) supported.
-  TypeMirror reflectType(Type type);
+  TypeMirror? reflectType(Type type);
 
   /// Returns a mirror of the given library [library].
-  LibraryMirror findLibrary(String library);
+  LibraryMirror? findLibrary(String library);
```

Comment:
Can we document when this might return null?

Reply:
It can't. Changed the return type to non-nullable.